include/graphblas/hyperdags/blas2.hpp

```diff
@@ -565,9 +569,13 @@ namespace grb {
 			return vxm< descr >( u, mask, v, A, ring, phase );
 		}
 		const RC ret = vxm< descr >(
-			internal::getVector(u), internal::getVector(mask),
-			internal::getVector(v), internal::getVector(v_mask), internal::getMatrix(A),
-			ring, phase
+			internal::getVector(u),
+			internal::getVector(mask),
```

Comment:
I think it's clearer if these two are grouped (and similar further down for u and v_mask--- I agree the matrix could/should go on its own row)